cni/pkg/repair/repaircontroller.go

```diff
@@ -62,23 +63,36 @@ func NewRepairController(reconciler brokenPodReconciler) (*Controller, error) {
 					fieldSelectors = append(fieldSelectors, fs)
 				}
 			}
+			// filter out pod events from different nodes
+			fieldSelectors = append(fieldSelectors, fmt.Sprintf("spec.nodeName=%v", reconciler.cfg.NodeName))
```

Comment:
is this going to be an issue if they have already defined this in `reconciler.cfg.FieldSelectors`? Seems like its actually ok to have multiple duplicate selector although its a bit weird

Reply:
Yeah it should be ok to have duplicated selectors.